### PR TITLE
[Configuration] Update overview after 1st draft 

### PR DESCRIPTION
**Pull request by ThomDietrich**
The first version of the article looks nice but was written in the middle of the night and *does* contains errors. Please review and be kind 😴 Maintainers: Please edit directly.

Signed-off-by: Thomas Dietrich <Thomas.Dietrich@tu-ilmenau.de>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
@kaikreuzer @cdjackson @ghys please check the Comparison table. This is just a first draft. Please let me know where your respective parts need a tick or a cross and which other tasks you would like to add as noteworthy. Thanks ;)

http://docs.openhab.org/configuration/index.html

File: configuration/index.md [hunk 1]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
       <td>❌</td>
       <td>✔️</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Define Sitemaps</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
       <td>(HABPanel)</td>
```
Comment (kaikreuzer) [inline configuration/index.md]: X here (HABPanel cannot do it)

File: configuration/index.md [hunk 2]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
       <td>❌</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Access Sitemaps</td>
       <td>❌</td>
       <td>✔️ (limited)</td>
```
Comment (kaikreuzer) [inline configuration/index.md]: X here

File: configuration/index.md [hunk 3]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
+  </tbody>
 </table>
 
 {: #text-vs-ui}
 ## Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only. <!-- Draft -->
-One of the most visible additions to openHAB 2 was the adminstrative web interface Paper UI. <!-- Draft -->
+With openHAB 1.x the smart home configuration was done via configuration files only.
+One of the most visible additions to openHAB 2 was the adminstrative web interface Paper UI.
```
Comment (kaikreuzer) [inline configuration/index.md]: " was the" -> "is the"

File: configuration/index.md [hunk 4]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
 
-Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database. <!-- Draft -->
-Both methodes can be used side-by-side or even mixed, e.g. a Thing is discovered by Paper UI and connected Items are defined in a Items configuration file. <!-- Draft -->
+Things and Items can either be defined and managed in configuration files or handled by Paper UI in a system-side database.
+Both methodes can be used side-by-side or even mixed, e.g. a Thing is discovered by Paper UI and connected Items are defined in a Items configuration file.
```
Comment (kaikreuzer) [inline configuration/index.md]: typo: methods

File: configuration/index.md [hunk 5]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
-<table class="centered highlight"> <!-- Draft -->
+<table class="centered highlight">
   <thead>
-    <tr> <!-- Draft -->
+    <tr>
       <th data-field="task">Task</th>
       <th data-field="files">Textual Configuration</th>
       <th data-field="paperui">Paper UI</th>
       <th data-field="habmin">HABmin</th>
       <th data-field="uis">other UIs</th>
```
Comment (kaikreuzer) [inline configuration/index.md]: I would actually remove the "other UIs" column

File: configuration/index.md [hunk 6]
```diff
@@ -7,151 +7,151 @@ title: Configuration
...
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Modify openHAB Settings</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>❌</td>
```
Comment (kaikreuzer) [inline configuration/index.md]: tick here

File: configuration/index.md (hunk 1 above)
Comment (ghys) [inline configuration/index.md]: Indeed!

Comment (ThomDietrich) [inline configuration/index.md]: Indeed yes or indeed no? :) I'd want to mention the special functionality of HABPanel, gladly as it's separate row.

File: configuration/index.md (hunk 5 above)
Comment (ThomDietrich) [inline configuration/index.md]: why? removing them will shadow their existence. If any we should brake it down to a counted list. I could also add these as a tooltip or smaller subtitle.

Comment (kaikreuzer) [inline configuration/index.md]: As we are within the configuration category, I would only list UIs that are meant for configuration, which are Paper UI and HABmin. With that logic, I would even remove HABPanel from here.

Comment (ThomDietrich) [inline configuration/index.md]: Ahh. that's a strong argument :) deal.

**Comment by ThomDietrich** [comment]
@cdjackson good to go?

File: configuration/index.md (hunk 1 above)
Comment (ghys) [inline configuration/index.md]: Sorry, indeed no it cannot do it :)

To be exhaustive, the terminology currently used in HABPanel is:
At the top you have a "panel registry", stored as a JSON object inside a configuration property which can contain several "panel configurations", each being a set of "dashboards" (and other settings, widget definitions etc.)... and each dashboard contains "widgets" :)

So I guess you could say it defines and accesses "panel configurations" or "dashboards", but this concept is currently specific to HABPanel, and cannot be easily defined or accessed outside of it; though as stated above the panel registry is a configuration variable so there are ways to alter it using Paper UI (_Configuration > Services > UI > HABPanel > Show more_) or the `config` subshell of the Karaf commandline (`config:edit org.openhab.habpanel` and so on), but it's definitely not recommended.

Comment (ThomDietrich) [inline configuration/index.md]: I've removed the column as agreed upon with Kai. Would be great to see more details like these in the README aka. http://docs.openhab.org/addons/uis/habpanel/readme.html ;)

Comment (ghys) [inline configuration/index.md]: @ThomDietrich yes I saw this just after commenting. I'd be willing to document the more advanced/technical concepts of HABPanel, how do you envision it?
Can I add a few Markdown files in the repo (there is already a `doc` folder there), link to them in the main README.md and they will end up being part of documentation? If so what about the TOC?

Comment (ThomDietrich) [inline configuration/index.md]: Ah I see, yes you can do that just as you already do with the images. We can copy more markdown documents from your repo.
Regading markdown and tricks you can use with docs, check other articles ;)

Comment (ghys) [inline configuration/index.md]: Got it, thanks!

**Pull request merged**